model/DataBaseAccess.php

```diff
@@ -177,7 +178,28 @@ public function addMultiplePermissions(array $permissionData)
                 }
             }
         }
-        $this->getPersistence()->insertMultiple(self::TABLE_PRIVILEGES_NAME, $insert);
+
+        $this->getLogger()->debug(
+            "Processing {count} permission inserts in {chunks} chunks",
+            [
+                'count' => count($insert),
+                'chunks' => ceil(count($insert)/self::INSERT_CHUNK_SIZE)
+            ]
+        );
+
+        foreach(array_chunk($insert, self::INSERT_CHUNK_SIZE) as $index => $batch) {
+            $this->getLogger()->debug(
+                "Processing chunk {index}/{total} with {items} ACL entries",
```

Comment:
Please, use single quotes.
When you are using double quotes - you are checking the string for possible variables.

Reply:
Very true :facepalm: Changed in 39763796a360382af747a8883cc311b13dfe8ddc, thanks for spotting this :+1: